### PR TITLE
react-props-and-expressions- Marianne Cuevas

### PR DESCRIPTION
**Pull request by mariannecuevas**
Screenshot 2023-07-18 154530

### CODE REVIEW

File: react-props-and-expressions/my-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import './App.css';
+
+function CustomButton(props) {
```
Comment (dseongdev) [inline react-props-and-expressions/my-app/src/App.js]: Try extracting this component out to its own file and look into props destructuring.

**Review by dseong0224 — CHANGES_REQUESTED** [review]
left a comment

**Review by dseong0224 — APPROVED** [review]

File: react-props-and-expressions/my-app/src/CustomButton.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function CustomButton({ text, color }) {
+  const style = {
+    backgroundColor: color,
+  };
+
```
Comment (dseongdev) [inline react-props-and-expressions/my-app/src/CustomButton.js:5]: depending on how long your variables get you can decide to inline style as well. 

`return <button style={{
    backgroundColor: color
  }}>{text}</button>;`

**Review by dseong0224 — APPROVED** [review]

File: react-props-and-expressions/my-app/src/CustomButton.js (hunk 2 above)
Comment (mariannecuevas) [reply react-props-and-expressions/my-app/src/CustomButton.js:5]: Oh! Okay I was wondering about that. Thank you!

**Pull request merged**